src/backend/web/validation.js

```diff
@@ -50,4 +51,41 @@ function validateNewFeed() {
   };
 }
 
-module.exports.validateNewFeed = validateNewFeed;
+// query validation starts here
+// queryValidation rules
+const queryValidationRules = () => {
```

Comment:
Fold this into the `validateQuery` function below, so we don't expose it outside this module.

Reply:
Incorporated in [7dd761b](https://github.com/Seneca-CDOT/telescope/pull/1334/commits/7dd761bc7b9281cd31a572a30424e0c41235cb6b)